test/e2e/storage/volume_metrics.go

```diff
@@ -103,7 +103,15 @@ var _ = utils.SIGDescribe("[Serial] Volume metrics", func() {
 
 		Expect(len(updatedStorageMetrics)).ToNot(Equal(0), "Error fetching c-m updated storage metrics")
 
-		volumeOperations := []string{"volume_provision", "volume_detach", "volume_attach"}
+		volumeOperations := []string{
+			"volume_provision",
+			"volume_detach",
+			"volume_attach",
+			"volume_manager_total_volumes",
+			"volume_manager_reconstruct_volume_errors_total",
+			"attachdetach_controller_total_volumes",
+			"attachdetach_controller_total_forced_detaches",
+		}
```

Comment:
Is adding this key enough? How do we ensure that this operation does indeed occur within the test? Have you ran this test?